backend/Services/CoordinateFetcher.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Services;
+
+use GuzzleHttp\Client;
+use Internships\Models\FetchAddress;
+
+class CoordinateFetcher
+{
+    protected string $mapboxToken;
+
+    public function __construct()
+    {
+        $this->mapboxToken = $_ENV["MAPBOX_TOKEN"];
```

Comment:
Is there any package to manage .env files? I think we should avoid use globals.

Reply:
There's dotenv which loads those from provided .env file into those.

Is there some better way to manage them?

Reply:
Check if there is `env()` function.
Otherwise we can use `getenv()` or stay with `$_ENV[]`.
https://www.php.net/manual/en/function.getenv.php

Reply:
It seems env() is not an available option, and getenv() has no access to the values from .env file.

It seems php is not configured to use them and it gains access only through dotenv.

Reply:
`getenv()` should has access to the $_ENV global. Are you sure that it is not working?

Reply:
Yeah, for whatever reason it returns false for this property, the direct access into $_ENV works and script executes in an expected way.